### PR TITLE
Fix service tree selection remove focus problem

### PR DESCRIPTION
**Pull request by Dremars**
- Added focus to title after removing all selections
- Added focus to previous or next elements on selection list

### CODE REVIEW

**Comment by Dremars** [comment]
[(2) Palveluluettelon fokusointi ongelma](https://trello.com/c/zffZZlb3/752-2-palveluluettelon-fokusointi-ongelma)

File: src/views/ServiceTreeView/ServiceTreeView.js [hunk 1]
```diff
@@ -155,6 +157,31 @@ const ServiceTreeView = (props) => {
...
+    handleCheckboxClick(e, item);
+    // If focus set to true
+    // Attempt to focus to either previous or next sibling in list
+    if (focus) {
+      const sibling = e.currentTarget.parentNode?.previousSibling?.childNodes[1]
+        || e.currentTarget.parentNode?.nextSibling?.childNodes[1];
+      if (sibling) {
+        sibling.focus();
+      } else if (titleRef.current) {
+        console.log('titleRef', titleRef);
```
Comment (mikkoisa) [inline src/views/ServiceTreeView/ServiceTreeView.js:171]: Forgot console.log here

**Review by mikkoisa — APPROVED** [review]
Otherwise looks good! Tested on iOS and Mac VoiceOver

File: src/views/ServiceTreeView/ServiceTreeView.js (hunk 1 above)
Comment (Dremars) [reply src/views/ServiceTreeView/ServiceTreeView.js]: Fixed in d6de0c1

**Pull request merged**